doc/docs/modules/ROOT/pages/producer.adoc

```diff
@@ -1,38 +1,38 @@
+= Source: Neo4j -> Kafka
```

Comment:
Does this title needs the prefix "Neo4j Streams -" like the other two chapters about Sink and Procedures modules?

Reply:
No it shouldn't need the prefix - having the prefix doesn't really add any value to the page